projects/canopy/src/lib/promo-card/promo-card/promo-card-image/promo-card-image.component.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../../../styles/mixins';
+@import '../../../../styles/variables/components/promo-card';
+
+.lg-promo-card-image__content {
+  width: 100%;
+  height: 20rem;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+
+  @media (min-width: $promo-card-inline-breakpoint) {
+    width: 21.5rem;
```

Comment:
Does this 21.5 need to stay in sync with the one here? https://github.com/Legal-and-General/canopy/pull/219/files#diff-2b27734b611e9da58de61c6ae166d208c3d986131b49342df51ba5ec9cdb56b3R8
Just wondering if these should become css variables?

Reply:
yes, I'll extract these to a variable.